swap/src/asb/config.rs

```diff
@@ -1,6 +1,7 @@
 use crate::fs::{ensure_directory_exists, system_config_dir, system_data_dir};
 use crate::tor::{DEFAULT_CONTROL_PORT, DEFAULT_SOCKS5_PORT};
 use anyhow::{bail, Context, Result};
+use bdk::bitcoin::util::bip32::ExtendedPubKey;
```

Comment:
We also depend on `bitcoin`, no need to import things through the re-export. Please prefer a direct import from the `bitcoin` crate unless you want to be uniform across the codebase and import via bdk everywhere. Then we could remove the `bitcoin` dependency but I think we will need it anyway to activate certain features.